README.md

```diff
@@ -183,33 +180,32 @@ wakePin <- hardware.pinW;
 wakePin.configure(DIGITAL_IN_PULLDOWN, readBuffer);
 
 // Configure accelerometer
-accel.init();
 accel.setDataRate(100);
 
 // Configure the FIFO buffer in Stream Mode and set interrupt generator
 // to generate an interrupt when there are 30 entries in the buffer
-accel.configureFifoInterrupt(true, LIS3DH_FIFO_STREAM_MODE, 30);
+accel.configureFifoInterrupt(true, accel.FIFO_STREAM_MODE, 30);
```

Comment:
This doesn't look right - this should be a constant not a class variable?

Reply:
@smittytone can you please look into this one?

Reply:
Sure